Classes/Typo3/RefIndex.php

```diff
@@ -163,7 +162,7 @@ protected function updateTable($tableName)
                     ->where(
                         $queryBuilder->expr()->eq(
                             'tablename',
-                            $queryBuilder->createNamedParameter($tableName, PDO::PARAM_STR)
+                            $queryBuilder->createNamedParameter($tableName, Connection::PARAM_STR)
```

Comment:
Why switching to the `Connection::PARAM_STR` instead of `PDO::PARAM_STR`
in Docs in e.g. here done with `PDO::PARAM_INT` , so would expect to stay with `PDO::PARAM_STR` too. 
https://docs.typo3.org/m/typo3/reference-coreapi/10.4/en-us/ApiOverview/Database/QueryBuilder/Index.html#where-andwhere-and-orwhere

Reply:
In some cases the sysext extensions use PDO, in others Connection:
https://github.com/TYPO3/TYPO3.CMS/blob/10.4/typo3/sysext/scheduler/Classes/Scheduler.php#L278
I can change it back to PDO.